stumpy/stamp.py

```diff
@@ -59,20 +61,22 @@ def mass(Q, T, M_T, Σ_T, trivial_idx=None, excl_zone=0, left=False, right=False
         if D[:trivial_idx].size:
             IL = np.argmin(D[:trivial_idx])
             PL = D[IL]
-        if zone_start <= IL < zone_stop:
-            IL = -1
+            if PL == np.inf or zone_start <= IL < zone_stop:
```

Comment:
Maybe this is just how it is showing up on Github but this looks over-indented.

Reply:
Actually this indentation should be ok, because if we are outside the `if D[:trivial_idx].size` bracket, then IL is -1 anyways. But we can remove one indent if you say it improves readability.

Reply:
Yes, please un-indent. This will help reduce the complexity of the nested if conditions (two deep is okay, three deep likely means we should re-think the logic)